pep-0001.txt

```diff
@@ -293,14 +294,17 @@ the time when when Python was led by `a BDFL <Python's BDFL_>`_.
 Any legacy references to "BDFL-Delegate" should be treated as equivalent to
 "PEP-Delegate".
 
-Individuals offering to serve as PEP-Delegate should notify the Steering
-Council, PEP authors and PEP sponsor of their intent to self-nominate.
+An individual offering to nominate themselves as a PEP-Delegate must notify
+the relevant authors and (when present) the sponsor for the PEP, and submit
+their request to the Steering Council
+(which can be done via a `new issue <Steering Council issue_>`_ ).
```

Comment:
```suggestion
(which can be done via a `new issue <Steering Council issue>`_ ).
```

I don't believe the first underscore in the link target is required.

I'd also probably say ``request to the Steering Council through a `new issue <...>`_`` but at this point marginal benefit is quickly decreasing, so I won't formally suggest it!

Reply:
> I don't believe the first underscore in the link target is required.

It is here, since otherwise it is read as a literal URL rather than a target name and the link breaks (unlike `:ref:`, `:doc:`, etc directives that use an otherwise very similar syntax where it isn't required, since the content of `<>` must be an implicit or explicit target name, not a literal URL).

> I'd also probably say request to the Steering Council through a `new issue <...>`_ but at this point marginal benefit is quickly decreasing, so I won't formally suggest it!

I also preferred that wording and originally implemented something very close to it, but ultimately deferred to adhering pretty closely to what @brettcannon suggested above.